.vscode/settings.json

```
@@ -26,6 +26,7 @@
     },
     "python.formatting.provider": "yapf",
     "[json]": {
-	    "editor.formatOnSave": false
-    }
+        "editor.formatOnSave": false
+    },
+    "cSpell.words": ["EUCALYN", "HENK", "MHEN"]
```

We wouldnt really want this
```suggestion
	    "editor.formatOnSave": false
    }
```